src/chromecast.cr

```diff
@@ -59,6 +71,9 @@ class Castblock::Chromecast
 
           begin
             message = WatchMessage.from_json(output)
+            if message.payload && (payload_data = WatchMessagePayload.from_json(message.payload.as(String)))
```

Comment:
Nice hack :D

The thing is `from_json` can fail if the payload is not the expected message. This should probably not arrive, but just to be sure I think we could wrap the code in a begin / rescue block like this
```suggestion
            message.payload.as?(String).try do |payload|
              begin
                message.payload_data = WatchMessagePayload.from_json(payload)
              rescue ex
                Log.debug { "Unhandled payload: #{ex}" }
              end
            end
```

I also used `as?` which returns `nil` if it is not a string, and `try` which executes the given block only if the value is not `nil`.